receiver/hostmetricsreceiver/internal/scraper/networkscraper/network_scraper.go

```diff
@@ -113,69 +115,46 @@ func (s *scraper) scrapeAndAppendNetworkCounterMetrics(metrics pdata.MetricSlice
 	if len(ioCounters) > 0 {
 		startIdx := metrics.Len()
 		metrics.EnsureCapacity(startIdx + networkMetricsLen)
-		initializeNetworkPacketsMetric(metrics.AppendEmpty(), metadata.Metrics.SystemNetworkPackets, startTime, now, ioCounters)
-		initializeNetworkDroppedPacketsMetric(metrics.AppendEmpty(), metadata.Metrics.SystemNetworkDropped, startTime, now, ioCounters)
-		initializeNetworkErrorsMetric(metrics.AppendEmpty(), metadata.Metrics.SystemNetworkErrors, startTime, now, ioCounters)
-		initializeNetworkIOMetric(metrics.AppendEmpty(), metadata.Metrics.SystemNetworkIo, startTime, now, ioCounters)
+
+		s.recordNetworkPacketsMetric(now, ioCounters)
+		s.recordNetworkDroppedPacketsMetric(now, ioCounters)
+		s.recordNetworkErrorPacketsMetric(now, ioCounters)
+		s.recordNetworkIOMetric(now, ioCounters)
+		s.mb.Emit(metrics)
```

Comment:
@abhide looks like we still have an extra `Emit` call. Please remove it

Reply:
Sorry. missed this.. Fixed